src/App.tsx

```diff
@@ -34,62 +34,88 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  if (sortType === 1) {
+    visibleGoods.sort((a, b) => a.localeCompare(b));
+  } else if (sortType === 2) {
```

Comment:
`=== number` is not a good idea when you have an enum
apply this for the whole code

```suggestion
  if (sortType === SortType.ALPHABET) {
    visibleGoods.sort((a, b) => a.localeCompare(b));
  } else if (sortType === SortType.LENGTH) {
```